src/main/java/org/broadinstitute/hellbender/utils/bigquery/vetSchema.csv

```diff
@@ -0,0 +1,35 @@
+reference_name, STRING, NULLABLE, Reference name.
+start_position, INTEGER, NULLABLE, Start position (0-based). Corresponds to the first base of the string of reference bases.
+end_position, INTEGER, NULLABLE, End position (0-based). Corresponds to the first base after the last base in the reference allele.
+reference_bases, STRING, NULLABLE, Reference bases.
+alternate_bases, RECORD, REPEATED, One record for each alternate base (if any).
+alternate_bases.alt, STRING, NULLABLE, Alternate base.
+alternate_bases.AC, INTEGER, NULLABLE, Allele count in genotypes for each ALT allele in the same order as listed
+alternate_bases.AF, FLOAT, NULLABLE, Allele Frequency for each ALT allele in the same order as listed
+names, STRING, REPEATED, Variant names (e.g. RefSNP ID).
+quality, FLOAT, NULLABLE, Phred-scaled quality score (-10log10 prob(call is wrong)). Higher values imply better quality.
+filter, STRING, REPEATED, List of failed filters (if any) or "PASS" indicating the variant has passed all filters.
+call, RECORD, REPEATED, One record for each call.
+call.name, STRING, NULLABLE, Name of the call.
+call.genotype, INTEGER, REPEATED, Genotype of the call. "-1" is used in cases where the genotype is not called.
```

Comment:
At some point we should keep track of all these edge cases the variant transforms code deals with and see what we get for free from converting records to [VariantContext](https://github.com/samtools/htsjdk/blob/master/src/main/java/htsjdk/variant/variantcontext/VariantContext.java). This would be a discussion with David Roazen or Laura.  Also a lower priority for sure.